Text layer issue

Hello,

I am running version 1.7.2 and am experiencing an issue with text-layer, the text is rendering below the pdf preview.

Text layer on:
![Screenshot 2023-09-05 at 1 00 44 PM](https://github.com/TaTo30/VuePDF/assets/50152606/c9ed1561-d882-419f-a943-d195797d0f99)

Text layer off:
![Screenshot 2023-09-05 at 1 00 33 PM](https://github.com/TaTo30/VuePDF/assets/50152606/56c839a1-a317-423e-b2a5-0ca0d04e6f45)

If thats the expected behavior, then so be it but i recall in 1.5.1 it behaving differently. Thank you anyhow for taking a look into this for me.

Regards,
Jose

Hey Aldo, 

Thanks for the help, I didn't notice that change. However, with that import I still encounter an issue.

```
import "@tato30/vue-pdf/style.css";
```
Results in this error ```Module not found: Error: Package path ./style.css is not exported from package /Users/jose/Documents/GitHub/ERP/node_modules/@tato30/vue-pdf (see exports field in /Users/jose/Documents/GitHub/ERP/node_modules/@tato30/vue-pdf/package.json)```

I modified @tato30/vue-pdf package.json exports object based on the error log above and it works:
```
  "exports": {
    ".": {
      "require": "./dist/index.umd.js",
      "import": "./dist/index.mjs",
      "types": "./dist/types/index.d.ts"
    },
    "./*.css": "./dist/*.css",                     <---- I added this line
    "./**/*.css": "./dist/*.css",
    "./src/*": "./src/*"
  },
```

Let me know if I should provide more information or what I can do to make text layers work without package.json modification. Again I appreciate the help!

Jose

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@tato30/vue-pdf",
-  "version": "1.7.1",
+  "version": "1.7.4",
   "description": "PDF viewer for Vue 3",
   "author": {
     "name": "Aldo Hernandez",
@@ -26,7 +26,7 @@
       "import": "./dist/index.mjs",
       "types": "./dist/types/index.d.ts"
     },
-    "./**/*.css": "./dist/*.css",
+    "./*.css": "./dist/*.css",
     "./src/*": "./src/*"
   },
   "main": "./dist/index.umd.js",

